### PR TITLE
Support major version forwarding

### PR DESCRIPTION
**Pull request by hsong-rh**
Support major version forwarding.

Reference: https://github.com/ManageIQ/service_portal-api/pull/109

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@ def add_swagger_route(http_method, path, opts = {})
 
     full_path = path.gsub(/{(.*?)}/, ':\1')
     scope :as => :api, :module => "api", :path => prefix do
+      match "/v0/*path", :via => [:delete, :get, :options, :patch, :post], :to => redirect(:path => "/#{prefix}/v0.0/%{path}", :only_path => true)
```
Comment (gmcculloug) [inline config/routes.rb]: I think the idea was to redirecting to the latest minor version, redirecting to `0.0` was meant to be a temporary change.  cc @syncrou

Comment (syncrou) [reply config/routes.rb]: Correct, we will be defaulting to v0.1.0 on service_portal after the resource routes are merged ( as a separate PR. )

Comment (hsong-rh) [reply config/routes.rb]: Ok, `v0.0` is the only available version so far in approval service. Will update to `v0.1.0` when it's needed.

Comment (bzwei) [reply config/routes.rb]: Isn't it `0.0.1` with last PR(#47) merged?

Comment (hsong-rh) [reply config/routes.rb]: @bzwei `0.0.1` is the version for openapi.json. I think we only care the major and minor versions so far.

**Pull request merged**